packages/lib/sanitizeCalendarObject.ts

```diff
@@ -4,7 +4,6 @@ export const sanitizeCalendarObject = (obj: DAVObject) => {
   return obj.data
     .replaceAll("\r\n", "\r")
     .replaceAll("\r", "\r\n")
-    .replaceAll(/(: \r\n|:\r\n|\r\n:|\r\n :)/gm, ":")
```

Comment:
This caused 
`URL;VALUE=URI:\r\nEND:VEVENT`
to turn into 
`URL;VALUE=URI:END:VEVENT`

This doesn't seem like much but it basically destroys the END of VEVENT by pulling it up instead of in a new line which is essential for the ICAL parser to know that that's the end of the VEVENT. 🤯 

So, 
```
URL;VALUE=URI:END:VEVENT
```
is what it becomes whereas it should have been 
```
URL;VALUE=URI:
END:VEVENT
```